server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -81,17 +94,25 @@ async def fetch_releases():
     data = await asyncio.gather(
         fetch_prs(), fetch_comments(), fetch_commit_authors(), fetch_commit_committers(),
         fetch_reviews(), fetch_releases())
-    users = {}
+
+    stats = defaultdict(dict)
     for rows, key in zip(data, ("prs", "commenter", "commit_author", "commit_committer",
                                 "reviewer", "releaser")):
         for row in rows:
-            users.setdefault(row[0], {})[key] = row[1]
-    try:
-        del users[None]
-    except KeyError:
-        pass
+            stats[row[0]][key] = row[1]
+
+    keys = ["login", "email", "avatar_url", "name"]
     user_details = await db.fetch_all(
-        select([User.login, User.avatar_url, User.name])
-        .where(User.login.in_(users)))
-    user_details = {r[0]: (r[1], r[2]) for r in user_details}
-    return users, user_details
+        select([getattr(User, k) for k in keys])
+        .where(User.login.in_(stats.keys())))
+
+    indexed_keys = list(enumerate(keys))
+    contribs = []
+    for ud in user_details:
+        c = {k: ud[i] for i, k in indexed_keys}
```

Comment:
```suggestion
        c = {k: d for k, d in zip(keys, ud)}
```
so you don't have to materialize `list(enumerate(keys))`

Reply:
I bumped into this as it will work on sqlite, but not with pg. The problem is that iter on `ud` with sqlite will iterate on values while on pg it will iterate on `keys` (col names).